geomesa-cassandra/geomesa-cassandra-datastore/src/test/scala/org/locationtech/geomesa/cassandra/data/CassandraDataStoreTest.scala

```diff
@@ -0,0 +1,134 @@
+/***********************************************************************
+  * Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+  * All rights reserved. This program and the accompanying materials
+  * are made available under the terms of the Apache License, Version 2.0
+  * which accompanies this distribution and is available at
+  * http://www.opensource.org/licenses/apache2.0.php.
+  *************************************************************************/
+
+package org.locationtech.geomesa.cassandra.data
+
+import java.io.File
+
+import com.datastax.driver.core.Cluster
+import com.vividsolutions.jts.geom.Coordinate
+import org.cassandraunit.CQLDataLoader
+import org.cassandraunit.dataset.cql.ClassPathCQLDataSet
+import org.cassandraunit.utils.EmbeddedCassandraServerHelper
+import org.geotools.data.simple.SimpleFeatureStore
+import org.geotools.data.{DataStore, DataStoreFinder, DataUtilities}
+import org.geotools.factory.CommonFactoryFinder
+import org.geotools.feature.simple.SimpleFeatureBuilder
+import org.geotools.geometry.jts.JTSFactoryFinder
+import org.joda.time.DateTime
+import org.junit._
+import org.locationtech.geomesa.utils.geotools.Conversions._
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+
+class CassandraDataStoreTest  {
+
+  @Test
```

Comment:
Yes, starting the embedded cassandra was not working well with specs2.  So just going with vanilla Junit only for the cassandra store.